gaudi/README.md

```diff
@@ -0,0 +1,84 @@
+# Setting up HabanaAI Operator
+
+## Overview
+The steps mentioned in this document to deploy and configure [Habana AI Operator](https://catalog.redhat.com/software/container-stacks/detail/64342b3bcbfbb9a6588ce8dd) on Red Hat OpenShift are consolidated and referenced from [HabanaAI Operator for OpenShift](https://docs.habana.ai/en/latest/Orchestration/HabanaAI_Operator/index.html).
+
+## Prerequisities
+- Provisioned RHOCP cluster. Follow steps [here](/README.md).
+
+## Setting up OpenShift Environment
+- Setup Machine Configuration to update Firmware search path using below command. Please note that this step will reboot the nodes.
+```
+oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/gaudi/gaudi_firmware_path.yaml
+```
+ 
+- Setup Node Feature Discovery (NFD) instance using below command:
+```
+oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/gaudi/gaudi_nfd_instance_openshift.yaml
+```
+Verify NFD has labelled the node correctly:
+```
+oc get no -o json | jq '.items[].metadata.labels' | grep pci-1da3
+
+"feature.node.kubernetes.io/pci-1da3.present": "true",
+```
+NFD discovers underlying Habana Gaudi using its PCI device class and the vendor. Here `"feature.node.kubernetes.io/pci-1da3.present": "true"`, represents node has PCI card from the vendor 1da3 which is HabanaLabs. 
+
+## Install HabanaAI Operator on Red Hat OpenShift
+### Installation via web console
+Follow the steps below to install Intel Device Plugins Operator using OpenShift web console:
+1.	In the OpenShift web console, navigate to **Operator** -> **OperatorHub**.
+2.	Search for **HabanaAI Operator** in all items field -> Click **Install**.
+### Verify Installation via web console
+1.	Go to **Operator** -> **Installed Operators**.
+2.	Verify that the status of the operator is **Succeeded**.
+
+### Installation via command line interface (CLI)
+Apply the [install_operator.yaml](/device_plugins/install_operator.yaml) file:
+```
+oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/gaudi/gaudi_install_operator.yaml
+```
+
+### Verify Installation via CLI
+Verify that the operator controller manager pod is up and running:
+```
+oc get pod -n habana-ai-operator
+
+NAME                                  READY   STATUS    RESTARTS   AGE
+controller-manager-6c8459d9cb-fqs8h   2/2     Running   0          25m
+```
+
+## Resources Provided by Habana Gaudi Device Plugin
```

Comment:
Before this paragraph, should we add some more detail for user to check whether the provisioning is proper? Since the step above only shows that the operator is up and running, we do not know whether the OOT driver and device plugin has been loaded. OR we can just hide these detail and have a simple way to check the ready of provisioning? like check the number of   habana.ai/gaudi resources? 

And this paragraph should be move the end of the doc.

Reply:
This is just the installation of the operator. OOT drivers and gaudi device plugin is not yet created. 
It is in the following section - Creating Habana AI Operator DeviceConfig Instance.

Reply:
Please move the resource table to the end of the doc